openapi.yaml

```diff
@@ -0,0 +1,787 @@
+openapi: '3.0.3'
+info:
+  description: This is a simple API to manage object permissions
+  version: 1.0.4
+  title: Permissions Framework API
+  contact:
+    email: apanin@dominodatalab.com
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+tags:
+  - name: "securables"
+    description: "Securables (Objects that can be secured)"
+  - name: "permissionsets"
+    description: "Permission Sets (Permissions mapped to Subjects)"
+  - name: "grants"
+    description: "Grants (Permission Sets applied to Securables)"
+  - name: "checks"
+    description: "Checking permissions"
+  - name: "subjects"
+    description: "Authorization Subjects"
+  - name: "users"
+    description: "Users as Authorization Subjects"
+  - name: "groups"
+    description: "User Groups as Authorization Subjects"
+paths:
+  /permission-sets:
+    get:
+      operationId: listPermissionSets
+      tags:
+        - "permissionsets"
+      parameters:
+        - $ref: "#/components/parameters/pagingOffset"
+        - $ref: "#/components/parameters/pagingLimit"
+      responses:
+        "200":
+          description: "Display Permission Sets"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/PermissionSetsResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "500":
+          $ref: "#/components/responses/ServerError"
+    post:
+      operationId: createPermissionSet
```

Comment:
These are fixed now, correct?  It is not possible to create permission sets anymore?

Reply:
Permissions are fixed. Permission Sets group existing permissions.